src/pages/embedded/dev_env/platformio.md

```diff
@@ -10,8 +12,11 @@
 ```Json
 lib_deps = Luos
 board = <board name>
+-DLUOSHAL= <MCU family>
 ```
-Replace `<board name>` with the name of the board you are using, eg. `board = l0` for the L0 board.
+Replace `<board name>` with the name of the board you are using, eg. `board = l0` for the L0 board or `board = disco_f030r8` for the STM32 F0308 Discovery board
+
+Replace `<MCU family>` with the family of the MCU you are usign, eg. `-DLUOSHAL=STM32F0` for the F0 family.
```

Comment:
Replace `<LuosHAL folder>` with the <a href="https://github.com/Luos-io/LuosHAL">Luos HAL</a> folder corresponding to the family of the MCU you are usign, eg. `-DLUOSHAL=STM32F0` for the F0 family.